rdkshellimage.cpp

```diff
@@ -635,6 +701,230 @@ namespace RdkShell
         return true;
     }
 
+    void Image::BlendOver(unsigned char **rows_destination, unsigned char **rows_source, unsigned int x, unsigned int y, unsigned int width, unsigned int height)
```

Comment:
Please ensure the method names are camel case starting with a lower case letter to keep consistency with the other RDKShell APIs.